lib/configs/recommended.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   extends: require.resolve('./base.js'),
   rules: {
+    'nuxt/no-img-element': 'error',
```

Comment:
Sometimes it would be needed to test with plain <img> tag and if eslint module is enabled built will fail while it is not that mandatory IMO. (PS: Image module is not stable yet to always advice)

Reply:
Fair point